controllers/auth.go

```diff
@@ -59,6 +59,11 @@ func AuthenticateHandler(c echo.Context) error {
 	}
 
 	if u.Username == username && u.ValidPassword(password) {
+		err := h.ValidCliVersion(c.Request())
+		if err != nil {
```

Comment:
as `err` is only used for the conditional, you should better be using something like:

```go
if err := h.ValidCliVersion(c.Request()); err != nil {
    ...
}
```